sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -1,22 +1,31 @@
 # Release History
 
 ## 5.1.0-beta.7 (Unreleased)
-### Breaking changes
+- We are now targeting the service's v3.1-preview.5 API as the default instead of v3.1-preview.4.
+
+### Features Added
+- Added property `disableServiceLogs` to all endpoints' options bag
+- Added support for `Sentiment Analysis` as an action type for the `beginAnalyzeActions` API.
+  
+### Breaking Changes
 - Removed `batch` keyword from the model names, `AnalyzeBatchActionsResult`, `AnalyzeBatchActionsOperationDetail`, 
   `AnalyzeBatchActionOptions` and the related method names, such as renamed `beginAnalyzeBatchActions` to `beginAnalyzeActions`.
 
+### Key Bug Fixed
+- Fixed `NullPointerException` for passing value `null` to options tasks in the `TextAnalyticsActions`.
+
 ## 5.1.0-beta.6 (2021-04-06)
-### Breaking changes
+### Breaking Changes
```

Comment:
@mssfang just double checking why the old headers in the changelog are being updated. Since they've already been released with the old headers, wouldn't it be potentially confusing for users?

Reply:
We have a new rule on formatting CHANGELOG.
I updated the file based on the new rules. And keep it consistency as one single file
### Feature Added
### Breaking Changes
etc.